### DISCUSSION TITLE
Determining your OSPO's budget?

### DISCUSSION THREAD

**Discussion by jdorfman** [todogroup/ospology#14] [OSPO Discussions]
Ideally, I would love it if you could share your actual budget breakdown but if that isn't possible, percentages would be just as great. Thank you in advance.

**Comment by caniszczyk** [comment]
I think this is going to be a tough one for many organizations to share and maybe you can extrapolate budget based on the size of the OSPO: https://github.com/todogroup/osposurvey/tree/master/2020#ospos-come-in-all-shapes-and-sizes

I'd love to include this question in next year's OSPO survey if you want to open up an issue here: https://github.com/todogroup/osposurvey

**Comment by jeffpaul** [comment]
Its more likely that folks would feel comfortable sharing rough percentage breakdown for how time is budgeted/spent versus actual dollar amounts.  If that info is helpful, then I can look to share that from my end.
